src/views/credits/credits.jsx

```diff
@@ -114,80 +117,22 @@ const Credits = () => (
                     <FormattedMessage id="credits.pastContributorsThanks" />
                 </p>
                 <p>
-                    Yusuf Ahmad,
-                    Ben Berg,
-                    Amos Blanton,
-                    Paula Bontá,
-                    Carl Bowman,
-                    Karen Brennan,
-                    Juanita Buitrago,
-                    Leo Burd,
-                    Gaia Carini,
-                    Jolie Castellucci,
-                    Karishma Chadha,
-                    Kasia Chmielinski,
-                    Michelle Chung,
-                    Shane Clements,
-                    Hannah Cole,
-                    Sayamindu Dasgupta,
-                    Zachary Deiman,
-                    Margarita Dekoli,
-                    Manuj Dhariwal,
-                    Shruti Dhariwal,
-                    Dave Feinberg,
-                    Champika Fernando,
-                    Linda Fernsel,
-                    Elizabeth Foster,
-                    Joan Fusco,
-                    Lily Gabaree,
-                    Stephanie Gayle,
-                    Chris Graves,
-                    Joel Gritter,
-                    Megan Haddadi,
-                    Kreg Hanning,
-                    Sean Hickey,
-                    Christina Huang,
-                    Tony Hwang,
-                    Abdulrahman Idlbi,
-                    Rupal Jain,
-                    Randy Jou,
-                    Paul Kaplan,
-                    Lily Kim,
-                    Tauntaun Kim,
-                    Saskia Leggett,
-                    DD Liu,
-                    John Maloney,
-                    Katelyn Mann,
-                    Tim Mickel,
-                    Amon Millner,
-                    Andrés Monroy-Hernández,
-                    Marian Muthui,
-                    My Nguyen,
-                    Lisa O&apos;Brien,
-                    Abisola Okuk,
-                    Kristin Osiecki,
-                    Sarah Otts,
-                    Carmelo Presicce,
-                    Tina Quach,
-                    Mitchel Resnick,
-                    Ricarose Roque,
-                    Natalie Rusk,
-                    Andrea Saxman,
-                    Ray Schamp,
-                    Eric Schilling,
-                    Jay Silver,
-                    Brian Silverman,
-                    Andrew Sliwinski,
-                    Tammy Stern,
-                    Lis Sylvan,
-                    Bryce Taylor,
-                    Hanako Tjia,
-                    Jaleesa Trapp,
-                    Moran Tsur,
-                    Claudia Urrea,
-                    Kathy Wu,
-                    Julia Zimmerman,
-                    Oren Zuckerman.
+                    Yusuf Ahmad, Ben Berg, Amos Blanton, Paula Bontá, Karen Brennan,
```

Comment:
I think we should go back to putting names one per line -- it makes for much clearer diffs. 

My guess is that this was an unintentional change, because the same change also removed a number of names from this list, I think unintentionally. The change may have started with an out of date copy of the file.